### DISCUSSION TITLE
Why is @babel/core-7.12 installed, but not used?

### DISCUSSION THREAD

**Discussion by lightmare** [babel/babel#13625] [Q&A]
It's installed as a devDependency for @babel/preset-env. The only reason for that I could find is in this test:
https://github.com/babel/babel/blob/86fae72e5b47d141018e795e6d41b7ef6fd7c07a/packages/babel-preset-env/test/regressions.js#L1
But that line imports the current version from workspace, not 7.12

**Comment by fedeci** [comment]
It should be fixed then :)

**Reply by lightmare** [reply]
What would be the appropriate fix? I don't quite understand what exactly the regression being tested was.
If I change the import to `@babel/core-7.12`, the test fails and the diff shows it transpiled arrow function — which according to the test comment, it shouldn't.
```diff
expect(received).toMatchInlineSnapshot(snapshot)

Snapshot name: `#12880 read the .browserslistrc file when using @babel/core < 7.13.0 1`

- Snapshot  - 1
+ Received  + 3

Math.pow(a, b);

- a => b;
+ (function (a) {
+   return b;
+ });
```
